Samples/src/OrderFinding/OrderFinding.qs

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT License.
 
 namespace Microsoft.Quantum.Samples.OrderFinding {
+	open Microsoft.Quantum.Arrays;
```

Comment:
It looks like there's a slight tabs vs spaces issue here...

Reply:
I tried a new IDE and didn't spot these. Fixed them now and checked all other changed files as well.